scripts/install.js

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env node
 
-var npmArgs = JSON.parse(process.env.npm_config_argv);
+var npmArgs = JSON.parse(process.env.npm_config_argv).original;
 
-if (npmArgs.original.indexOf('--bootstrap') === -1) {
+if (npmArgs.indexOf('--create') === -1 && npmArgs.indexOf('--bootstrap') === -1) {
```

Comment:
True, they might introduce bootstrap as well - at any rate, for now, we have a fallback :D